### PR TITLE
Fix Issue #9658

### PR DESCRIPTION
**Pull request by Sliptory**
Add doc to CSharpCompilation.ClassifyConversion method

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs [hunk 1]
```diff
@@ -1821,6 +1821,14 @@ internal bool MightContainNoPiaLocalTypes()
         // compilation, while the other method needs a bindings object to determine what bound node
         // an expression syntax binds to.  Perhaps when we document these methods we should explain
         // where a user can find the other.
+        /// <summary>
+        /// Classifies a conversion from <paramref name="source"/> to <paramref name="destination"/> according
+        /// to this compilation's programming language.
```
Comment (333fred) [inline src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs]: This is a C# compilation, there's no ambiguities around "this compilation's language". Please remove the qualifier.

File: src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs [hunk 2]
```diff
@@ -1821,6 +1821,14 @@ internal bool MightContainNoPiaLocalTypes()
         // compilation, while the other method needs a bindings object to determine what bound node
         // an expression syntax binds to.  Perhaps when we document these methods we should explain
         // where a user can find the other.
+        /// <summary>
+        /// Classifies a conversion from <paramref name="source"/> to <paramref name="destination"/> according
+        /// C# programming language.
```
Comment (333fred) [inline src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs]: This wasn't quite what I meant. `Classifies a conversion from <paramref name="source"/> to <paramref name="destination"/>.`

Comment (Sliptory) [reply src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs]: I understood. Late, but understood.

**Comment by 333fred** [comment]
Thanks for the contribution @Sliptory!

**Pull request merged**